servicepulse/install-servicepulse-in-iis.md

```diff
@@ -100,6 +100,28 @@ WARNING: The default configuration for ServiceControl only allows access to REST
 
 It is also recommended that the IIS web site be configured to use SSL if an authorization provider is used.
 
+### Configuring Reverse Proxy in a non-root directory
+
+Due to a bug in SignalR (as at Microsoft.AspNet.SignalR.JS version 2.2.0), usage of IIS as a reverse proxy in a virtual directory requires an additional URL Rewrite Rule on the `/api/` sub folder. This rule makes sure that SignalR uses the correct path when hosted within a virtual directory. This rule should look as follows: 
```

Comment:
@SimonCropp the issue is https://github.com/SignalR/SignalR/issues/3287, do you want to have a look and see how to best include the link in the text ?